### PR TITLE
[INLONG-10927][SDK] Transform supports padding of left and right strings

### LINKED ISSUE #10927: [Feature][SDK] Transform supports padding of left and right strings
### Description

- parent issue #10796 

RPAD(s1,len,s2)：Fill string s2 at the end of string s1 to make the length of the string len
LPAD(s1,len,s2)：Fill string s2 at the beginning of string s1 to make the string length len

### Use case

```
RPAD('12',5,'34') -> "12343"
RPAD('1234',2,'5') -> "12"
RPAD('1',5,'') -> ""
RPAD(null,2,'34') or RPAD('12',null,'34') or RPAD('12',2,null) or RPAD('12',-1,'34') -> null

LPAD('12',5,'34') -> "34312"
LPAD('1234',2,'5') -> "12"
LPAD('1',5,'') -> ""
LPAD(null,2,'34') or LPAD('12',null,'34') or LPAD('12',2,null) or LPAD('12',-1,'34') -> null
```

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Zkplo**
Fixes #10927 

### Motivation

Add two function classes: RpadFunction and LpadFunction. Also, add the corresponding unit test codes.

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [x] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### CODE REVIEW

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/LpadFunction.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import org.apache.inlong.sdk.transform.process.parser.ValueParser;
+
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.Function;
+
+import java.util.List;
+
+/**
+ * LpadFunction
+ * description: Lpad(s1,len,s2) : Fill string s2 at the beginning of string s1 to make the string length len
```
Comment (yfsn666) [inline inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/LpadFunction.java:32]: It better to explain what the function returns when `len` is less than the length of `s1` in the description.

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/RpadFunction.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import org.apache.inlong.sdk.transform.process.parser.ValueParser;
+
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.Function;
+
+import java.util.List;
+
+/**
+ * RpadFunction
+ * description: RPAD(s1,len,s2) : Fill string s2 at the end of string s1 to make the length of the string len
```
Comment (yfsn666) [inline inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/RpadFunction.java:32]: ditto.

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/LpadFunction.java (hunk 1 above)
Comment (Zkplo) [reply inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/LpadFunction.java]: Thank you for your suggestion. More descriptions have been added, please check.

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/RpadFunction.java (hunk 2 above)
Comment (Zkplo) [reply inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/RpadFunction.java:32]: done

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/LpadFunction.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            return null;
+        }
+        int len = Integer.parseInt(OperatorTools.parseString(lengthObj));
+        if (len < 0) {
+            return null;
+        }
+        String rightStr = OperatorTools.parseString(rightStringObj);
+        if (rightStr.isEmpty()) {
+            return "";
+        }
```
Comment (yfsn666) [inline inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/LpadFunction.java:66]: Why this function return "" if s2 is "", is there any official documentation? For example, lpad("hi", 1, "") returns "h" may be more appropriate?

Comment (Zkplo) [reply inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/LpadFunction.java:72]: Thank you for your inspection. The official MySQL documentation does not provide a detailed description of it. This is my negligence, as I did not consider the situation where len is less than s1 length during the testing process. Now modifications have been made to class comments and code logic, please check.

**Review by yfsn666 — APPROVED** [review]

**Review by aloyszhang — DISMISSED** [review]

**Review by dockerzhang — DISMISSED** [review]

**Review by luchunliang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**